.tekton/pipeline-service-test.yaml

```
@@ -317,7 +317,7 @@ spec:
 
                 echo "Start executing pipeline cases ..."
                 TEST_DIR=$(find "$PWD" -type f -name test.sh -exec dirname {} +)
-                KUBECONFIG=/kubeconfig CASES=pipelines $TEST_DIR/test.sh
+                KUBECONFIG=/kubeconfig CASES=pipelines, chains $TEST_DIR/test.sh
```

```suggestion
                KUBECONFIG=/kubeconfig CASES=pipelines,chains $TEST_DIR/test.sh
```
Ah, there is an extra space.

---

Fixed